lib/cliff.js

```diff
@@ -19,20 +19,15 @@ cliff.__defineGetter__('logger', function () {
 
 cliff.__defineSetter__('logger', function (val) {
   logger = val;
-
-  //
-  // Setup winston to use the `cli` formats
-  //
-  if (logger.cli) {
-    logger.cli();
-  }
 });
 
 //
 // Set the default logger for cliff.
 //
-cliff.logger = new winston.Logger({
-  transports: [new winston.transports.Console()]
+cliff.logger = new winston.createLogger({
```

Comment:
```suggestion
cliff.logger = winston.createLogger({
```

Reply:
Oops you're right, I'll add it in a87d5dab0e6e213c7b42667ac804293f4393f726, thanks!